core/src/test/java/org/infinispan/commands/module/TestModuleLifecycle.java

```diff
@@ -59,7 +58,7 @@ public void cacheManagerStarting(GlobalComponentRegistry gcr, GlobalConfiguratio
 
       Set<String> componentNames = testGlobalConfiguration.globalTestComponents().keySet();
       for (String componentName : componentNames) {
-         AssertJUnit.assertSame(testGlobalConfiguration.globalTestComponents().get(componentName), gcr.getComponent(componentName));
+         assert testGlobalConfiguration.globalTestComponents().get(componentName) == gcr.getComponent(componentName);
```

Comment:
This should not depend on the test framework or will needlessly propagate testng dependency in pure junit tests.

Reply:
There's no such thing as a pure JUnit test depending on core test-jar :)

Reply:
in that case, bleahhhhhh :nauseated_face: